src/backend/api/handlers/team.py

```diff
@@ -65,6 +66,16 @@ def team(team_key: TeamKey, model_type: Optional[ModelType] = None) -> Response:
     return profiled_jsonify(team)
 
 
+def team_history(team_key: TeamKey) -> Response:
+    track_call_after_response("team/history", team_key)
+
+    events = TeamEventsQuery(team_key=team_key).fetch_dict(ApiMajorVersion.API_V3)
+    awards = TeamAwardsQuery(team_key=team_key).fetch_dict(ApiMajorVersion.API_V3)
+
+    history: History = History(events=events, awards=awards)
```

Comment:
I don't think the explicit `: History` is necessary because the type can be deduced automatically. I think?